test/es6/unscopablesWithScopeTest.js

```diff
@@ -349,7 +349,7 @@ var tests = [
                 assert.areEqual(3, c, "c is only on the proto");
                 a = 3;
                 b = 4;
-                assert.areEqual(2,proto.b,"proto.b is never set b\c child b is unnscopable");
+                assert.areEqual(2,proto.b,"proto.b is never set b\c child b is unscopable");

```

Comment:
Sure, fwiw, you can see the individual changes in https://github.com/jsoref/ChakraCore/commits/spelling-full :
`because`: jsoref@de4074a (that spelling is used a bunch)
adding `is`: jsoref@8bd0db8 (and one pair of those also omitted is)